package/package.json

```diff
@@ -44,17 +44,17 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.12.0 || ^5.0.0"
+		"astro": "^4.16.0 || ^5.0.0"
```

Comment:
Looks like the requirement is 4.14, according to https://docs.astro.build/en/reference/integrations-reference/#injecttypes-option